inference-engine/src/gna_plugin/gna_plugin_internal.hpp

```diff
@@ -53,6 +53,15 @@ class GNAPluginInternal  : public InferenceEngine::InferencePluginInternal {
         return make_executable_network(std::make_shared<GNAExecutableNetwork>(modelFileName, plg));
     }
 
+    ExecutableNetwork ImportNetwork(std::istream& networkModel,
```

Comment:
should not `ImportNetwork(const std::string &modelFileName ...` reuse this implementation which works with stream?